lib/average.js

```diff
@@ -0,0 +1,36 @@
+class Average {
+  total(array) {
+    return array.reduce((x, y) => { return x + y}, 0)
+  }
+  mean(array) {
+    return this.total(array) / array.length
+  }
+  median(array) {
+    return array[array.length / 2]
+  }
+  mode(array) {
+    let modeHash = {}
```

Comment:
Two things here:

1. In JS, the key-value data structure like a Ruby hash is called an object. Avoid referring to objects as hashes, even though they are very similar.

2. Avoid naming variables with their data type in them (e.g., `nameString` or `modeHash`).